app/models/generation.rb

```diff
@@ -0,0 +1,49 @@
+# generation.csv displays database
+# i.e. all groups that should have a generation-value must be included. For groups that are not included generation is going to be set nil
+
+class Groups < ActiveRecord::Base
+
+    def self.import_from_csv(file)
+        data = Buzzn::Utils::File.read(file)
+        check = []
+
+        CSV.parse(data.gsub(/\r\n?/, "\n"), col_sep: ';', headers: true) do |row|
+            gen = row['Generation']
+            group = Group::Localpool.find(Hash[row.to_a]['ID'])
+
+            if group.nil?
```

Comment:
Nachdem der begin...rescue Block oben eingefügt wurde, benötigt man an dieser Stelle keinen Logger mehr. Für den Falls, dass group nil ist, würde in Zeile 16 auch eh ein Fehler geworfen werden:
NoMethodError: undefined method `not_found' for nil:NilClass